app/partials/header.html

```diff
@@ -1,8 +1,22 @@
 <div class="header">
-  <ul class="nav nav-pills pull-right">
-    <li class="active"><a href="#">Home</a></li>
-    <li><a href="#">About</a></li>
-    <li><a href="#">Contact</a></li>
-  </ul>
-  <h3 class="text-muted">PageTraining</h3>
+  <div class="row head">
+    <div class="col-md-1 logo"></div>
+    <div class="col-md-4 col-md-offset-3"><input type="text" class="search"></div>
```

Comment:
For the input please take a look at bootstrap's classes for inputs. A raw input looks very bad :p Bootstrap's styles are much more nicer